doc/BUILD.bazel

```diff
@@ -4,7 +4,7 @@ load("@io_bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
 
 _DOC_SRCS = [
     "experimental",
-    "xcodeproj.doc",
```

Comment:
I'm leaving the `.doc` logic since I'm changing `experimental` to `experimental.doc` in a followup PR.